### PR TITLE
Choose 'png' image compression in BqplotImageView by default

### PR DESCRIPTION
**Pull request by bmorris3**
## Description

Allow users to take advantage of PNG compression via `ImageGL` from https://github.com/glue-viz/bqplot-image-gl/pull/109. Bumping pin for `bqplot-image-gl`.

### CODE REVIEW

**Comment by maartenbreddels** [comment]
The visual change is ok, so we can put the hash somewhere to indicate this is good, this should be documented at https://pytest-mpl.readthedocs.io/en/stable/hash_mode.html

Do you want to try this @bmorris3, or should we ask @astrofrog ?

**Comment by dhomeier** [comment]
@Carifio24, @astrofrog, are there any tools to run the `deploy-reference-images` job for updating the reference hashes in glue-jupyter-visual-tests, and does one need special permissions for that?

**Comment by bmorris3** [comment]
I experimented a bunch, and settled on generating updated hashes with:
```bash
tox -e py311-test-visual -- --mpl-results-path=results
```
I've put the resulting hashes in the latest commit, I guess we'll see if that's ok?

**Comment by dhomeier** [comment]
The `scatter2d` images apparently have not changed at all.

**Comment by astrofrog** [comment]
@dhomeier the CircleCI job updates the reference images

File: glue_jupyter/tests/images/py311-test-visual.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "glue_jupyter.bqplot.image.tests.test_visual.test_contour_units[chromium]": "53ed6a698cd0fe879f92a7c428e14878e80b2640a8e09d69bb82c3847041bfb9",
-  "glue_jupyter.bqplot.scatter.tests.test_visual.test_visual_scatter2d[chromium]": "91fe92465a858548adcfba167b4402c61b927dd4da61dd1ed486f9fed84d5915",
-  "glue_jupyter.bqplot.scatter.tests.test_visual.test_visual_scatter2d_density[chromium]": "d3bd7a7ff60cff14154f50bd75c5b8821cbd25009b01600691ca35b568783c43"
-}
\ No newline at end of file
+  "glue_jupyter.bqplot.image.tests.test_visual.test_contour_units[chromium]": "19517fbbea156e9569a2e6367e926e9f301bb27e8d2f600b42cf4577aa9064d4",
+  "glue_jupyter.bqplot.scatter.tests.test_visual.test_visual_scatter2d[chromium]": "d8cd6c7f5d85975aaea515c0b0fd1c715ed0823ddd8e85d05d69255eefee0a0a",
+  "glue_jupyter.bqplot.scatter.tests.test_visual.test_visual_scatter2d_density[chromium]": "da3864b8e513e7a7b43d58172ae85a6a56180a91c1f5e0de9545ca56ac6d6cfe"
```
Comment (dhomeier) [inline glue_jupyter/tests/images/py311-test-visual.json:4]:
```suggestion
  "glue_jupyter.bqplot.scatter.tests.test_visual.test_visual_scatter2d[chromium]": "91fe92465a858548adcfba167b4402c61b927dd4da61dd1ed486f9fed84d5915",
  "glue_jupyter.bqplot.scatter.tests.test_visual.test_visual_scatter2d_density[chromium]": "d3bd7a7ff60cff14154f50bd75c5b8821cbd25009b01600691ca35b568783c43"
```

File: glue_jupyter/tests/images/py311-test-visual.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "glue_jupyter.bqplot.image.tests.test_visual.test_contour_units[chromium]": "53ed6a698cd0fe879f92a7c428e14878e80b2640a8e09d69bb82c3847041bfb9",
+  "glue_jupyter.bqplot.image.tests.test_visual.test_contour_units[chromium]": "19517fbbea156e9569a2e6367e926e9f301bb27e8d2f600b42cf4577aa9064d4",
```
Comment (dhomeier) [inline glue_jupyter/tests/images/py311-test-visual.json:2]:
```suggestion
  "glue_jupyter.bqplot.image.tests.test_visual.test_contour_units[chromium]": "94a99faec5500df042edde00e9cdef58e230d68e9a762c1cd0ed0499eb0de83f",
```
different matplotlib/bqplot versions on CircleCI?

**Comment by dhomeier** [comment]
> @dhomeier the CircleCI job updates the reference images

Did you enable that job somehow? I noticed one job that required approval before, but first wanted to fix the checksums, and now it seems to have disappeared again.

**Comment by astrofrog** [comment]
@dhomeier - the job to update the reference images only happens on main after a PR has been merged. All that is needed for a PR to pass is for the hashes to be correct.

**Review by dhomeier — COMMENTED** [review]
Thanks all!

**Pull request merged**